src/site/content/en/blog/idle-detection/index.md

```diff
@@ -188,6 +188,12 @@ You can see the various options in the video below.
   </figcaption>
 </figure>
 
+### Puppeteer support
+
+As of Puppeteer version&nbsp;5.3.1, you can
+[emulate the various idle states](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageemulateidlestateoverrides)
+to test the API over the DevTools Protocol.
```

Comment:
Over the DevTools Protocol is an implementation detail that might not be super interesting to people. Maybe omit that and focus on what this means for them? Maybe something like:

```suggestion
to programmatically test how your web app’s behavior changes.
```